### ISSUE TITLE
pywis-pubsub readme incomplete + error in API description

### ISSUE THREAD

**Issue by david-i-berry** [World-Meteorological-Organization/pywis-pubsub#67]
- The pywis-pubsub readme is currently incomplete and missing important details. Information on how to set the publisher username and password is missing. 

- additionally, there is an error in the description of publishing via the API:

```python
message = create_message(
        topic='foo/bar',
        content_type='application/x-bufr',
        url='http://www.meteo.xx/stationXYZ-20221111085500.bufr4', 
        identifier='stationXYZ-20221111085500', 
        datetime=datetime.now(timezone.utc),
        geometry=[33.8, -11.8, 123],
        metadata_id='x-urn:wmo:md:test-foo:htebmal2001',
        wigos_station_identifier='0-20000-12345',
        operation='update'
)
```
`datetime` should be `datetime_`.

**Comment by david-i-berry** [comment]
Running the first example from the readme (via docker):

```bash
docker exec publisher pywis-pubsub publish --topic origin/a/wis2/centre-id/data/core/weather \
      --config /pywis-pubsub/config/config.yml \
      -i stationXYZ-20221111085500 
      -u "http://subscriber:5000/openapi" 
      -g 33.8,-11.8,8.112 -w 0-20000-12345
```

gives the following output but no message is published on my local broker.

```
Unknown content type                                                                                                                                
Connected to broker mqtt://broker:1883
```
This is the dockerfile I'm using:

```dockerfile
FROM python:3.11-slim-bookworm

# Update / upgrade
RUN apt-get update && \
    apt-get upgrade

# install python dependencies
RUN python -m pip install pywis-pubsub
COPY ./config /pywis-pubsub/config
```

Checking the broker logs I have:

```logfile
broker     | 2024-08-16T11:34:03: New connection from 172.23.0.3:37047 on port 1883.
broker     | 2024-08-16T11:34:03: Client pywis-pubsub-335 disconnected, not authorised.
```

Mosquitto conf:

```
#persistence true
persistence_location /mosquitto/data/
log_dest file /mosquitto/log/mosquitto.log
log_dest stdout
log_timestamp_format %Y-%m-%dT%H:%M:%S
password_file /mosquitto/config/password.txt
max_queued_messages 10

# ACLs
acl_file /mosquitto/config/acl.conf

## MQTT Listener
listener 1883
protocol mqtt
```

ACL:

```
user everyone
topic read public/#
topic read origin/#

user publisher
topic readwrite origin/#
topic readwrite public/#
topic readwrite internal/#
topic readwrite fs/#
topic read $SYS/#
```

With the user publisher (and pwd) set in password.txt.

Based on the current documentation there it is unclear how to pass the password and user for the publish command.

My pywis-pubsub.conf is minimal:

```
# RFC1738 URL
broker: mqtt://broker:1883
# clean mqtt session on connection. Set to false to start from server saved offset
clean_session: true
# whether to verify data and message
verify_data: true
validate_message: true
```

**Comment by david-i-berry** [comment]
For the second problem in the readme, the relevant section is here:

https://github.com/wmo-im/pywis-pubsub/blob/89d0e87e83fa6690dc497ddfd54e48398675311f/README.md?plain=1#L173-L190

Looking at the code: https://github.com/wmo-im/pywis-pubsub/blob/89d0e87e83fa6690dc497ddfd54e48398675311f/pywis_pubsub/publish.py#L101

It should be `datetime_`.

**Comment by tomkralidis** [comment]
README fixed in #68 (thanks @david-i-berry).
